src/near2far.cpp

```diff
@@ -250,12 +250,15 @@ void dft_near2far::farfield_lowlevel(std::complex<double> *EH, const vec &x) {
     component c0 = component(f->vc); /* equivalent source component */
 
     vec rshift(f->shift * (0.5 * f->fc->gv.inva));
-    size_t idx_dft = 0;
-    LOOP_OVER_IVECS(f->fc->gv, f->is, f->ie, idx) {
-      IVEC_LOOP_LOC(f->fc->gv, x0);
-      x0 = f->S.transform(x0, f->sn) + rshift;
-      for (int i = 0; i < Nfreq; ++i) {
-        double freq = freq_min + i * dfreq;
+#ifdef HAVE_OPENMP
+#  pragma omp parallel for
+#endif
```

Comment:
For some reason, I am unable to compile with OpenMP via `--with-openmp` on my local machine using this `#HAVE_OPENMP` macro (i.e., the build is successful but the `examples/binary_grating_n2f.py` test uses just a single thread/process for the `get_farfields` calculation regardless of the value for `OMP_NUM_THREADS`). To fix this problem, I had to revert back to using just `#pragma omp parallel for` (i.e., removing the `#ifdef` and `#endif` lines) and `CXX="g++ -fopenmp"`. Perhaps this is related to the `openmp` -related changes in `configure.ac` in this PR?

Reply:
Whoop, I forgot to add `OPENMP_CXXFLAGS` to `CXXFLAGS` … try it again.